resources/views/forum/insert_plaatje.blade.php

```diff
@@ -0,0 +1,3 @@
+<script>
+insertPlaatje("{{$plaatje->uid}}");
```

Comment:
Dit voelt gek.

Reply:
Ik ben zelf ook niet helemaal voorstander van dit soort inline scripts, maar het is wel een beetje hoe de stek in elkaar zit. Om het op een andere manier te doen gaat er denk ik behoorlijk wat extra javascript in zitten en ik weet niet of het op die manier echt veel mooier wordt.

Reply:
Misschien is het beter om het mogelijk te maken voor een response om een callback functie te bevatten, misschien met argumenten. Dan hoef je niet alle js uit te voeren die je toevallig in `domUpdate` tegen komt.

bijv:
```json
{
  "cb": "insertPlaetje",
  "args": ["blah"]
}
```

Reply:
Ja, maar dat komt wel een beetje op het zelfde neer eigenlijk toch? Alleen met andere syntax. Bovendien wordt momenteel ook al gewoon nieuwe JavaScript uitgevoerd in domUpdate (bijvoorbeeld in ketzers).

Reply:
Het verschil is dat je nu kan whitelisten. En ik ben van mening dat we al die javascript er tzt uit moeten slopen.